dask_jobqueue/core.py

```diff
@@ -174,7 +178,6 @@ def __init__(
         extra=None,
         env_extra=None,
         log_directory=None,
-        walltime=None,
```

Comment:
Just one question about this removal: shouldn't we instead remove it from all JobqueueCluster impl?